### PR TITLE
API Commits Complete

### PR DESCRIPTION
**Pull request by ColinReinhart**

### CODE REVIEW

File: lib/esty_commist.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'httparty'
```
Comment (tjhaines-cap) [inline lib/esty_commist.rb:1]: Do we need this file in lib or is this leftover from just starting the logic and things got moved around?

**Review by tjhaines-cap — CHANGES_REQUESTED** [review]
Everything looks great, just not sure the file in lib is needed.

File: lib/esty_commist.rb (hunk 1 above)
Comment (ColinReinhart) [reply lib/esty_commist.rb]: Did not need any of that. I have deleted commist file

**Review by tjhaines-cap — APPROVED** [review]
looks good

**Pull request merged**